scanner/src/main/kotlin/Scanner.kt

```diff
@@ -192,12 +192,7 @@ class Scanner(
                 }.onFailure {
                     controller.addProvenanceResolutionIssue(
                         pkg.id,
-                        Issue(
-                            source = TOOL_NAME,
-                            severity = Severity.ERROR,
-                            message = "Could not resolve provenance for package '${pkg.id.toCoordinates()}': " +
-                                    it.collectMessages()
-                        )
+                        Issue(source = TOOL_NAME, severity = Severity.ERROR, message = it.collectMessages())
```

Comment:
Is `it` always thrown from line 101, or could the exception be coming from elsewhere?

Reply:
That's the only place (other exceptions thrown in this file are collected into this one).